docs/experiment.md

```diff
@@ -0,0 +1,8 @@
+# Experiment
+
+## Overview
+
+## Common Conventions
+
+## Functions
+### run
```

Comment:
I think we should walk through what's happening in `run` at a high level, though

Reply:
e.g. mention the core loop, how we call `generate_actions_for_timestep`, how we handle actions, etc